frappe/email/doctype/email_account/email_account.json

```diff
@@ -214,7 +214,7 @@
   },
   {
    "depends_on": "eval: doc.enable_incoming && !doc.use_imap",
-   "description": "Append as communication against this DocType (must have field \"Status\" and both \"Sender\" and \"Subject\" defined in the related doctype Email Settings)",
+   "description": "Append as communication against this DocType (must have fields, \"Sender\", \"Subject\")",
```

Comment:
```suggestion
   "description": "Append as communication against this DocType (must have fields: \"Sender\" and \"Subject\")",
```

Reply:
Do you think there is a possibility to add this sentence too without causing problems : 

"These fields can be defined in the email settings section of the appended doctype."

Reply:
sure - there should be no problems as such 👍